tests/integ_tests/tensor_search/integ_tests/test_search_relevance_cutoff_feature.py

```diff
@@ -841,7 +841,7 @@ def test_sort_and_relevance_cutoff_pagination(self):
         )
 
         page_2_sort_candidates = page_2_results["_sortCandidates"]
-        self.assertEqual(11, page_2_sort_candidates)
+        self.assertEqual(7, page_2_sort_candidates)
```

Comment:
Let's confirm with Owen if this change is expected or not. I feel like we may need to return the true `sortCandidates`, instead of `sortCandidates - offset`.